.github/workflows/deprecate_release.yml

```diff
@@ -0,0 +1,34 @@
+name: deprecate_release
+
+on:
+  workflow_dispatch:
+    inputs:
+      commitHash:
+        required: true
+        type: string
+        description: The release commit hash that should be deprecated
```

Comment:
If this is referring to `releaseCommitHash` then both param name and description should include this context.